snap/snapcraft.yaml

```diff
@@ -12,7 +12,7 @@ apps:
   universal-ctags:
     command: ctags
     plugs:
-      - home
```

Comment:
You probably still want to specify the `home` interface so that ctags may access source files in your home folder. Removing this interface will prevent ctags from reading any files other than those exposed by the `personal-files` interface (see next comment)